language-sql.md

```diff
@@ -5,3 +5,10 @@
 ### Minimum Requirements
 
 MySQL keywords in uppercase.
+
+### Database table naming convention
+
+Prefix application name to table name. E.g.: `app1_users`, `app2_log_meta`.
+
+If the table is for multiple applications (normally this should be avoided), it's ok to leave it without the application name prefix.
```

Comment:
Normally we consider one database for one application, here's a good explanation: http://programmers.stackexchange.com/a/105788